README.md

```diff
@@ -31,6 +31,8 @@ provider, as it utilizes off-the-shelf components to establish encrypted tunnels
 Note that Submariner is in the **pre-alpha** stage, and should not be used for production purposes. While we welcome usage and
 experimentation, it is quite possible that you could run into bugs.
 
+Submariner is a Cloud Native Computing Foundation sandbox project.
```

Comment:
```suggestion
Submariner is a Cloud Native Computing Foundation (CNCF) Sandbox project.
```

Reply:
The sentence specifically has to be “We are a Cloud Native Computing Foundation sandbox project.” or a variant thereof; we can’t add “(CNCF)” or capitalise “sandbox”. See https://github.com/cncf/foundation/blob/master/website-guidelines.md

Reply:
Thanks for the link. I still find it odd that "sandbox" is not capitalised (it is in many other paces in the CNCF website), but these guidelines are very clear.

Reply:
Yeah it's weird, in the legal-focused docs it's not capitalized but everywhere else it is.